src/components/AddHouseForm/AddHouseForm.module.css

```diff
@@ -0,0 +1,55 @@
+.formContainer {
+  display: flex;
+  flex-direction: column;
+  max-width: 400px;
+  gap: 5px;
```

Comment:
Unfortunately, using `gap` with `flex` won't work on Safari 13, which is quite popular apparently. You don't have to change it but it's something to consider in the future.

Reply:
On Safari 15 it works :D So what do you suggest instead of it?